src/workbench/SideBarHeader.tsx

```diff
@@ -48,15 +50,29 @@ const SideBarHeader = () => {
       },
       [search, isUnmounted]
     );
+  const handleClear = useCallback(() => {
+    setQuery("");
+    setResults(undefined);
+  }, [setQuery, setResults]);
   // Width of the sidebar tabs. Perhaps we can restructure the DOM?
-  const offset = horizontalLogoRef.current
-    ? horizontalLogoRef.current.getBoundingClientRect().left
+  const sidebarWidth = useRef<HTMLDivElement>(null);
+  const offset = faceLogoRef.current
+    ? faceLogoRef.current.getBoundingClientRect().right + 14
     : 0;
-  const width = ref.current
-    ? ref.current!.clientWidth - offset - 14 + "px"
+  const width = sidebarWidth.current
+    ? sidebarWidth.current!.clientWidth - offset - 14 + "px"
     : undefined;
+
+  const { sizedPaneSize } = useSplitViewContext();
+  const [searchWidth, setSearchWidth] = useState<number | undefined>();
+  useEffect(() => {
+    setSearchWidth((sizedPaneSize || 0) - offset - 14);
+  }, [offset, sizedPaneSize]);
```

Comment:
We need a better way than this. It causes so much lag on moving the divider.

The problem I was trying to solve was having a long search query when the search dialog is closed. Avoid text or it's container spilling over into the editor. Avoid this happening when the user moves the split view divider. I think we need a `maxWidth` from somewhere. Putting `overflow="hidden"` in the wrong place also hides the focus ring.

Reply:
I think I just pushed a fix for it, which was simple, and I was overcomplicating it.